modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-cart-test/src/testIntegration/java/com/liferay/headless/commerce/delivery/cart/resource/v1_0/test/CartResourceTest.java

```diff
@@ -74,6 +81,26 @@ public void setUp() throws Exception {
 
 		_commerceChannel = CommerceTestUtil.addCommerceChannel(
 			testGroup.getGroupId(), _commerceCurrency.getCode());
+
+		_country = _countryLocalService.addCountry(
+			"XY", "XYZ", true, true, RandomTestUtil.randomString(),
+			RandomTestUtil.randomString(), RandomTestUtil.randomString(),
+			RandomTestUtil.nextDouble(), true, true, false, _serviceContext);
+
+		_region = _regionLocalService.addRegion(
+			_country.getCountryId(), true, RandomTestUtil.randomString(),
+			RandomTestUtil.nextDouble(), RandomTestUtil.randomString(),
+			_serviceContext);
+
+		_cartAddress = _addressLocalService.addAddress(
```

Comment:
can you move this directly in the tests like we are doing in _testPatchCartByExternalReferenceCodeWithMoreExternalReferenceCodes with the Address randomAddress? And call it randomAddress